src/test/java/org/kohsuke/github/Github2faTest.java

```diff
@@ -0,0 +1,32 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+public class Github2faTest  extends AbstractGitHubWireMockTest {
+
+	@Test
+	public void test2faToken() throws IOException {
+		//assertFalse("Test only valid when not proxying", mockGitHub.isUseProxy());
```

Comment:
```suggestion
		assumeFalse("Test only valid when not proxying", mockGitHub.isUseProxy());
```
The test will be skipped when proxying, but will run in CI.

Reply:
fixed